doc/api/quic.md

```diff
@@ -1417,6 +1417,7 @@ added: REPLACEME
     `object.passphrase` if provided, or `options.passphrase` if it is not.
   * `activeConnectionIdLimit` {number}
   * `maxAckDelay` {number}
+  * `maxActiveCIDLimit` {number}
```

Comment:
Should we follow the ngtcp2 naming and spell `ConnectionID` out here? I think I’d be in favour of doing that, it’s a bit more intuitive

Reply:
Yeah I'm good with that